deps/verifier/Cargo.toml

```diff
@@ -49,6 +49,7 @@ strum.workspace = true
 veraison-apiclient = { git = "https://github.com/chendave/rust-apiclient", branch = "token", optional = true }
 ear = { git = "https://github.com/veraison/rust-ear", rev = "43f7f480d09ea2ebc03137af8fbcd70fe3df3468", optional = true }
 x509-parser = { version = "0.14.0", optional = true }
+reqwest = { version="0.12.9", features = ["blocking"] }
```

Comment:
I you are already specifying the needed version of `reqwest` in the global Cargo file, you can just say `request = workspace.true` here.

Reply:
Changed. If appropriate, please mark this comment as Resolved.

Reply:
See Ding's suggestions about the Cargo file as well.